pom.xml

```diff
@@ -24,12 +24,19 @@
   </licenses>
 
   <scm>
-    <connection>scm:git:git@github.com:cloudbees/super-simple-disk-usage-plugin.git</connection>
-    <developerConnection>scm:git:git@github.com:cloudbees/super-simple-disk-usage-plugin.git</developerConnection>
-    <url>http://github.com/cloudbees/super-simple-disk-usage-plugin</url>
+    <connection>scm:git:git@github.com:jenkinsci/cloudbees-disk-usage-simple.git</connection>
+    <developerConnection>scm:git:git@github.com:jenkinsci/cloudbees-disk-usage-simple.git</developerConnection>
+    <url>https://github.com/jenkinsci/cloudbees-disk-usage-simple</url>
     <tag>HEAD</tag>
   </scm>
 
+  <distributionManagement>
+    <repository>
+      <id>maven.jenkins-ci.org</id>
+      <url>http://maven.jenkins-ci.org/content/repositories/releases/</url>
```

Comment:
@tfennelly I'm going to check it. There are a lot of plugins using `distributionManagement` in its `pom.xml`.